start-site/src/main/java/io/spring/start/site/extension/dependency/springcloud/SpringCloudContractStubRunnerMavenBuildCustomizer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.dependency.springcloud;
+
+import io.spring.initializr.generator.buildsystem.Dependency;
+import io.spring.initializr.generator.buildsystem.maven.MavenBuild;
+import io.spring.initializr.generator.language.Language;
+import io.spring.initializr.generator.language.kotlin.KotlinLanguage;
+import io.spring.initializr.generator.project.ProjectDescription;
+import io.spring.initializr.generator.spring.build.BuildCustomizer;
+import io.spring.initializr.generator.version.Version;
+import io.spring.initializr.generator.version.VersionReference;
+
+/**
+ * {@link BuildCustomizer} for projects containing Spring Cloud Contract Stub Runner built
+ * with Maven.
+ *
+ * @author Eddú Meléndez
+ */
+class SpringCloudContractStubRunnerMavenBuildCustomizer implements BuildCustomizer<MavenBuild> {
+
+	private static final Version VERSION_2_2_0_M2 = Version.parse("2.2.0.M2");
+
+	private final ProjectDescription description;
+
+	SpringCloudContractStubRunnerMavenBuildCustomizer(ProjectDescription description) {
+		this.description = description;
+	}
+
+	@Override
+	public void customize(MavenBuild mavenBuild) {
+		if (isKotlin(this.description.getLanguage()) && isSpringCloudVersionAtLeastAfter()) {
```

Comment:
Hmm... it's required for the plugin dependency or in general? Cause maybe `spring-cloud-kotlin-spec` should provide this dependency if it doesn't.

Reply:
I am not sure about having it at `spring-cloud-kotlin-spec` since it just fails for the consumer and not the producer. This is the message thrown when try to run the consumer without it.

```
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchStubRunner' defined in class path resource [org/springframework/cloud/contract/stubrunner/spring/StubRunnerConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.contract.stubrunner.BatchStubRunner]: Factory method 'batchStubRunner' threw exception; nested exception is java.lang.NoClassDefFoundError: org/jetbrains/kotlin/scripting/compiler/plugin/repl/GenericReplCompiler
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.contract.stubrunner.BatchStubRunner]: Factory method 'batchStubRunner' threw exception; nested exception is java.lang.NoClassDefFoundError: org/jetbrains/kotlin/scripting/compiler/plugin/repl/GenericReplCompiler
Caused by: java.lang.NoClassDefFoundError: org/jetbrains/kotlin/scripting/compiler/plugin/repl/GenericReplCompiler
```